app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,58 @@
+class TasksController < ApplicationController
+  before_action :set_project, only: %i[index new show create destroy edit update]
+  before_action :set_task, only: %i[show edit update destroy]
+
+  def index
+    @tasks = @project.tasks.all
```

Comment:
`All` is redundant here
```suggestion
    @tasks = @project.tasks
```